fbuapp/View Controllers/HomeViewController.m

```diff
@@ -20,22 +20,28 @@
 #import "Favorite.h"
 #import "AppDelegate.h"
 #import "FilterViewController.h"
+#import "PillCell.h"
+#import "Query.h"
 
 @interface HomeViewController () <PostCellDelegate, PostTableViewDelegate, FilterDelegate, CategoryHeaderViewDelegate, UIScrollViewDelegate>
 
 @property (strong, nonatomic) PFQuery *postQuery;
-@property PostTableView * feed;
+@property PostTableView *feed;
+@property (strong, nonatomic) CategoryHeaderView *pillSelector;
+@property (strong, nonatomic) Query *savedQuery;
 
 @end
 
 
 @implementation HomeViewController
 
-- (void)viewDidLoad {
+- (void)viewDidLoad{
     [super viewDidLoad];
     self.feed.numberOfPosts = 0;
     self.postQuery = [Post query];
-    self.feed = [[PostTableView alloc] initWithUserId:@"myuserid"];
+    self.savedQuery = [[Query alloc] init];
+    
+    self.feed = [[PostTableView alloc] initWithUserId:@"myuserid"];//[try current user here]
```

Comment:
Does this need to be set to the current user's objectId?

Reply:
Yes this should but I never had current user working on my machine so it hasn't been tested yet